kansha/card_addons/members/models.py

```diff
@@ -5,4 +5,52 @@
 # This software is licensed under the BSD License, as described in
 # the file LICENSE.txt, which you should have received as part of
 # this distribution.
-#--
\ No newline at end of file
+#--
+
+from nagare import database
+from sqlalchemy import func
+from elixir import ManyToOne, ManyToMany, using_options
+
+from kansha.models import Entity
+from kansha.card.models import DataCard
+from kansha.user.models import DataUser
+from kansha.column.models import DataColumn
+
+
+class DataMembership(Entity):
+    using_options(tablename='user_cards__card_members')
+
+    user = ManyToOne(DataUser, primary_key=True)
+    card = ManyToOne(DataCard, primary_key=True)
+
+    @classmethod
+    def get_for_card(cls, card):
+        return cls.query.filter_by(card=card)
+
+    @staticmethod
+    def favorites_for(card):
+        query = database.session.query(DataUser)
```

Comment:
Not so huge in fact: at worst, it will return all the board members (each unique). I didn't find a simple and fast way to exclude the users already affected to the card at the query level, so I couldn't put the limit here. Do you have an idea ?